service/src/main/java/com/ericsson/eiffel/remrem/generate/controller/RemremGenerateController.java

```diff
@@ -199,7 +196,7 @@ public ResponseEntity<?> generate(final String msgProtocol, final String msgType
                         "Invalid JSON format,expected either single template or array of templates",
                         JSON_ERROR_STATUS, errorResponse);
             }
-        } catch (REMGenerateException | JsonSyntaxException e) {
+        } catch (REMGenerateException | JsonSyntaxException | NumberFormatException e) {
```

Comment:
NumberFormatException cannot be handled that way. It's something completely different then the other ones.